azurerm/resource_arm_iothub_test.go

```diff
@@ -100,9 +115,28 @@ resource "azurerm_iothub" "test" {
     capacity = "1"
   }
 
+  endpoint {
```

Comment:
can we split this out into it's own test?

Reply:
Agree - that will make it cleaner